webapi/_test_data/all_data/webern-data.ttl

```diff
@@ -0,0 +1,204 @@
+@prefix : <http://www.knora.org/data/08AE/webern#> .
+@prefix dc: <http://www.knora.org/ontology/dc#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix knora-base: <http://www.knora.org/ontology/knora-base#> .
+@prefix salsah-gui: <http://www.knora.org/ontology/salsah-gui#> .
+@prefix webern: <http://www.knora.org/ontology/08AE/webern#> .
+
+#
+# 
+# 
+# #################################################################
+# #
+# #    Individuals
+# #
+# #################################################################
+# 
+# 
+# http://data.knora.org/webern/testperson1
+
+<http://data.knora.org/webern/testperson1> a webern:Person, owl:NamedIndividual ;
```

Comment:
The correct data IRI would be: `http://rdfh.ch/08AE/testperson1`. Please see: http://www.knora.org/documentation/manual/rst/knora-api-server/api_v2/knora-iris.html#iris-for-data

Reply:
Thanks for the link.

Concerning UUIDs, the specs say:
```To keep data IRIs relatively short, each UUID is Base64 encoded, using the ‘URL and Filename safe Base64 Alphabet’ specified in Table 2 of RFC 4648, without padding; thus each UUID is a 22-character string.```

Could you point me to an UUID generator that generates those 22-character UUIDs. I can only find generators for much more (36) or less (20, 10) characters.

Reply:
When creating the resources by hand, I wouldn't worry too much, as long as the ones you make are unique. Just use what is easiest for you.

Reply:
ok.